### ISSUE TITLE
Shortcut-Feature issue with inline editable fields

### ISSUE THREAD

**Issue by KPeschke** [neos/neos-ui#2012]
### Description
Sometimes shortcuts are also triggered when focusing an inline editable field.

### Steps to Reproduce (not always)
1. create new page
1. set focus on the inline editable headline
2. press shortcut, e.g ti

#### Expected behavior
No shortcut triggering

#### Actual behavior
All toggle shortcuts could be triggered

### Affected Versions
Neos: 3.3.12
UI: 1.1.2

**Comment by mstruebing** [comment]
This shouldn't happen ofc, but there may be are some timing issues - we are thinking about introducing a modifier key or something but aren't sure at this point

**Comment by dimaip** [comment]
Does it still happen with the latest release?

**Comment by dimaip** [comment]
@mstruebing so this issue is no longer relevant and can be closed?

**Comment by dimaip** [comment]
Ah, I see, then we should find a reliable way to reproduce it and fix it

**Comment by mstruebing** [comment]
Yep, no one came up with a good solution by now, except modifier keys but someone said we should first check how other web apps are doing it and that was it until now

**Comment by kitsunet** [comment]
Can confirm, have this issue as well.

**Comment by dfeyer** [comment]
I can confirm too. Did we have a place in the state where we can store "currentlyEditing" when focusing on input and inline. And then the shortcut module can take decision based on this value ?

**Comment by dimaip** [comment]
@dfeyer Yes, we do.

**Comment by dfeyer** [comment]
> @dfeyer Yes, we do.

But it doesn't work in this case ?

**Comment by mstruebing** [comment]
This is the current implementation: https://github.com/neos/neos-ui/blob/9fa571adba1fb5631e5398886932634a1d043bc1/packages/neos-ui/src/Sagas/UI/Hotkeys/index.js#L39

I might have some time soonish to try another approach
